### PR TITLE
Add integration tests for MariaDB to CI

### PR DESCRIPTION
**Pull request by brfrn169**
This PR adds integration tests for MariaDB to the CI. Please take a look!

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -354,6 +354,38 @@ jobs:
           name: gradle_jdbc_sqlite_integration_test_reports
           path: core/build/reports/tests/integrationTestJdbc
 
+  integration-test-for-jdbc-mariadb:
+    runs-on: ubuntu-latest
+
+    services:
+      mariadb:
+        image: mariadb:10.11
```
Comment (brfrn169) [inline .github/workflows/ci.yaml:362]: Use the LTS version `10.11` of MariaDB for the integration test.

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Comment by brfrn169** [comment]
@Torch3333 I will merge this for now. Please take a look when you get a chance!

**Pull request merged**